src/mappings/burnedAssetsAmountHandler.ts

```diff
@@ -0,0 +1,15 @@
+import { SubstrateEvent } from "@subql/types";
+import { BurnedAssetsAmount } from "../types";
+
+export async function handleBurnedAssetsAmount(event: SubstrateEvent): Promise<void> {
+    logger.info(`event burned ${event.event.data}`);
+    const { event: { data: [ currencyId, , balance ] } } = event;
```

Comment:
Maybe add another attrubutes like` event.block.block.header.number` for desctructing here? Or not use destructing, to have one style in adding instance properties below

Reply:
It's better to keep current implementation due to consistency to others handlers.